task/thermal/src/control.rs

```diff
@@ -895,16 +962,21 @@ impl<'a> ThermalControl<'a> {
         Ok(())
     }
 
-    /// Attempts to set the PWM duty cycle of every fan in this group.
+    /// Attempts to set the PWM duty cycle of every present fan in this group.
```

Comment:
Clarify here that we _also_ try to set a PWM duty cycle for missing fans, but it's 0.